wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsWorkUnitsClient.java

```diff
@@ -888,6 +888,17 @@ public WorkunitInfo compileWUFromECL(WorkunitInfo wu ) throws Exception
             createdWU.setCluster(wu.getCluster());
             submitWU(createdWU); // if no exception proceed
             this.monitorWUToCompletion(createdWU);
+
+            //exceptions, etc. aren't always included in the submit response; do another request to get all workunit info
```

Comment:
We should prob. make use of WorkunitInfo getWUInfo(String wuid,boolean unarchive) here